### DISCUSSION TITLE
slow video

### DISCUSSION THREAD

**Discussion by vivienprouvot** [SubtitleEdit/subtitleedit#4922] [Ideas]
Hi, 

First of all, thank you for this wonderful program.

I'm looking for a way to slow video as I type what I hear.
I guess it was related to the video player integrated, let's say it's VLC.
I can't find any VLC menu inside the subtitle edit video part.

Does anybody know how I can do that if there is a way ?
If there isn't, should I add a ticket somewhere (in the issues maybe?)

Thank you  by advance ;)
Tingle

**Comment by niksedk** [comment]
Hi,

Below the waveform there's a double green arrow where it should be possible to adjust the speed:
![image](https://user-images.githubusercontent.com/1261331/112117415-a34fe700-8bbb-11eb-9379-019c741943ad.png)

Also, I would suggest not using VLC as the seeking is good - will cause a jumpy waveform (VLC 4 should fix this). "mpv" works well with SE.

**Reply by vivienprouvot** [reply]
> 
> 
> Hi,
> 
> Below the waveform there's a double green arrow where it should be possible to adjust the speed:
> ![image](https://user-images.githubusercontent.com/1261331/112117415-a34fe700-8bbb-11eb-9379-019c741943ad.png)
> 
> Also, I would suggest not using VLC as the seeking is good - will cause a jumpy waveform (VLC 4 should fix this). "mpv" works well with SE.

Thank you very much :) 
Synchronicities <3

**Reply by OmrSi** [reply]
Note that there is also a shortcut for it which you can set from the settings.

**Comment by vivienprouvot** [comment]
Well.... I found it, on bottom the double green arrows.
I prefer let my message in case someone ask the same thing.

Life's funny, I'm looking for it for months, and as soon as I ask, I magically try the button

**Reply by chris-der-kleine** [reply]
I think you could change the category though
